tests/abort1.rs

```diff
@@ -246,7 +246,7 @@ async fn test_stop_taker_after_setup() {
         .unwrap();
     let taker_balance = taker_balance_descriptor_utxo + taker_balance_swap_coins;
 
-    assert_eq!(org_taker_balance - taker_balance, Amount::from_sat(4227));
+    assert_eq!(org_taker_balance - taker_balance, Amount::from_sat(6768));
```

Comment:
How did this number change? Is it because of the new coinselection?

Reply:
I guess, though I couldn't do a thorough analysis, I am thinking of doing an in-depth analysis of this.

Reply:
Opened a GFI for this.

Reply:
https://github.com/utxo-teleport/teleport-transactions/issues/144